py4DSTEM/process/preprocess/electron_counting.py

```diff
@@ -211,7 +211,7 @@ def count_datacube(datacube,counted_shape,sigmathresh=4,nsamples=40,upperlimit=1
             #Put the electron_posn in fractional coordinates
             #where the positions are fractions of the original array
             electron_posn /= np.asarray([max(y,x),max(y,x)])[np.newaxis,:]
-            electron_posn = np.hstack((electron_pons
+            electron_posn = np.hstack((electron_posn
                            ,np.asarray([tt,xx],dtype=np.float32)[np.newaxis,:]))

```

Comment:
`xx` is also undefined here

Reply:
Hi Chris, `xx` should instead be `zz`: 

```suggestion
                           ,np.asarray([tt,zz],dtype=np.float32)[np.newaxis,:]))
```

The idea is that the code outputs a n_electrons x 4 numpy array, and for each electron the position in a single frame (y,x) is stored in the first two dimensions with the scan position (tt,zz) stored in the final two dimensions.